plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractBuildBootableJarMojo.java

```diff
@@ -811,8 +813,17 @@ private void setupPluginOptions() {
             if (pluginOptions.isEmpty()) {
                 pluginOptions = Collections.
                         singletonMap(Constants.OPTIONAL_PACKAGES, Constants.PASSIVE_PLUS);
-            } else if (!pluginOptions.containsKey(Constants.OPTIONAL_PACKAGES)) {
-                pluginOptions.put(Constants.OPTIONAL_PACKAGES, Constants.PASSIVE_PLUS);
+            } else {
+                if (!pluginOptions.containsKey(Constants.OPTIONAL_PACKAGES)) {
+                    pluginOptions.put(Constants.OPTIONAL_PACKAGES, Constants.PASSIVE_PLUS);
+                }
+                if (pluginOptions.containsKey(MAVEN_REPO_PLUGIN_OPTION)) {
+                    Path path = Paths.get(pluginOptions.get(MAVEN_REPO_PLUGIN_OPTION));
+                    if (path != null && !path.isAbsolute()) {
```

Comment:
Not really a big deal, but the path won't ever be `null` here. If there is a concern that `pluginOptions` contains a `null` value then an NPE would be thrown in the `Paths.get()`.

Also is a URL allowed here?

Reply:
Thank-you, yes... I am now checking for null map value. No URL, just file path.